actions/setup-nix/action.yml

```diff
@@ -104,6 +104,7 @@ runs:
         sudo ln -s ~/.aws $ROOT_PATH
 
     - uses: cachix/install-nix-action@8887e596b4ee1134dae06b98d573bd674693f47c # v26
+      if: inputs.enable-cachix == 'true'
```

Comment:
if we are adding this then we should create a changeset file for this action too

Reply:
Added.